samples/snippets/src/test/java/dialogflow/cx/DetectIntentIntentInputTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dialogflow.cx;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Unit test for {@link DetectIntentIntentInput}. */
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class DetectIntentIntentInputTest {
+
+  private static String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static String LOCATION = "global";
+  private static String AGENT_ID =
+      System.getenv()
+          .getOrDefault("DIALOGFLOW_CX_AGENT_ID_GLOBAL", "b8d0e85d-0741-4e6d-a66a-3671184b7b93");
+  private static String INTENT_ID = "00000000-0000-0000-0000-000000000000";
```

Comment:
Should these be real UUID's?  Remember you might have 9 of these running at the same time in different processes / processors.

Reply:
Each agent has a unique Agent ID that's created by Dialogflow. There was a permanent "TestAgent" resource created for the samples, and it has its own Flow IDs, Page IDs, etc. that are also created by Dialogflow (`00000000-0000-0000-0000-000000000000` is the Default Welcome Intent Flow ID).

Am I understanding the question correctly? I can go in and create a new intent by hand as well, if that makes more sense.